index.html

```diff
@@ -0,0 +1,156 @@
+              </div>
+                <div>
+                  <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                    <div>
+                        <button>html</button>
+                        <button>css</button>
+                        <button>javascript</button>
+                    </div><br>
+                      <div>
+                        <button class="btn-proj">See Project</button>
+                      </div>
+          <div class="container4">
+            <div>
+              <img class="portolio_img" src="forth_img.png" alt="portfolio image">
+            </div>
+              <div>
+                <p class="tittle">Tonic</p>
+              </div>
+                <div>
+                  <p class="words"><b class="canopy">CANOPY</b><img class="img" src="Counter.png" alt="counter">Back End Dev<img class="img" src="Counter.png" alt="counter">2015</p>
+                </div>
+                  <div>
+                    <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required </p>
+                  </div>
+                    <div>
+                        <button>html</button>
+                        <button>css</button>
+                        <button>javascript</button>
+                    </div><br>
+                      <div>
+                        <button class="btn-proj">See Project</button>
+                      </div>
+          </div>
+            <section class="about">
+       <h2 class="header">About <br> Myself</h2>
+          <p class="section">
+           I’m a software developer! I can help you build a product,feature or 
+           website Look through some of my work and experience! If you like what 
+           you see and have a project you need coded, don’t hestiate to contact me.
+          </p>
+           <p class="footer">LET CONNECT</p>
+            <div class="img">
+              <a href="#"><img src="twitter.svg" alt="twitter handle"></a>
+              <a href="#"><img src="github.jpg" alt="github handle"></a>
+              <a href="#"><img src="medium.jpg" alt="medium handle"></a>
+              <a href="#"><img src="angellist.jpg" alt="angellisthandle"></a>
+            </div>
+              <div>
+                <button class="btn-proj">Get my resume</button>
+              </div>
+              <br>
+                <div class="ellipse">
+                  <button><img src="Ellipse1.png" alt="ellipse"></button>
+                  <br>
+                  <button><img src="Ellipse2.png" alt="ellipse"></button>
+                  <br>
+                   <button><img src="Ellipse3.png" alt="ellipse"></button>
+                </div>
+                <div class="parag">
+                  <p>Frameworks</p>
+                  <p>Skills</p>
+                </div>
```

Comment:
- [ ]  Frameworks  and Skills are supposed to be links, thus, I suggest to add them into `<ul> <li> <a>` tags
```
<ul>
  <li><a href="#"><img src="#" alt="github handle">Frameworks</a></li>
  <li><a href="#"><img src="#" alt="github handle">Skills</a></li>
</ul>
```